frontend/src/scenes/dashboard/dashboardLogic.tsx

```diff
@@ -1479,6 +1490,51 @@ export const dashboardLogic = kea<dashboardLogicType>([
         },
     })),
 
+    subscriptions(({ values, actions }) => ({
+        variables: (variables) => {
```

Comment:
Should this not be `maybeUrlVariables`? Or is this triggered from somewhere else?

Reply:
this is sort of dicey basically maybeURLVariables needs to be set first on load and then when variables loads, it'll do a check the maybe variables to populate if possible (which is why this subscription is mounted on `variables`)